docs/tutorials/pytorch.ipynb

```diff
@@ -0,0 +1,507 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
```

Comment:
Would like to start formatting the copyright in code cells, as tools like `ruff` can only check code cells: https://github.com/astral-sh/ruff/issues/12802